test/buffer/api/api_test.go

```diff
@@ -33,88 +33,85 @@ func TestBufferApiE2E(t *testing.T) {
 		t.Skip("Skipping API E2E tests on Windows")
 	}
 
-	_, testFile, _, _ := runtime.Caller(0)
-	testsDir := filepath.Dir(testFile)
-	rootDir := filepath.Join(testsDir, "..", "..", "..")
```

Comment:
Toto je vypocet projektoveho adresara, podla aktualneho suboru.
Presunul som to do `NewRunner` a `CompileBinary` aby to nebolo zbytocne duplicitne v samotnych testoch.